From 303d07fe47d9c624e77cfa1316fc4141ac632e3f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 24 Aug 2021 19:58:12 +0300
Subject: [PATCH] [SW-36150] gaudi2: arm: make: use absolute path

Use absolute path by prepending the PREBOOT_ROOT_DIR variable.
This aligns to other Makefiles.

Change-Id: I4b11e63caf4129257fe9ae5a9d3cb64a1c5ce163
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/arm/Makefile b/arch/gaudi2/arm/Makefile
index a9ee777..838a8c4 100644
--- a/arch/gaudi2/arm/Makefile
+++ b/arch/gaudi2/arm/Makefile
@@ -3,7 +3,7 @@
 FW_SIZE := 0x4400
 PPBOOT_MAX_SIZE := 20480
 
-PREBOOT_SRCS_C += $(shell ls lib/preboot/thermal_v2.c)
+PREBOOT_SRCS_C += $(shell ls $(PREBOOT_ROOT_DIR)/lib/preboot/thermal_v2.c)
 
 CFLAGS += -mstrict-align
 image_validation: image_validation_common
